libs/scss-foundation/src/modules/typography/_fonts.scss

```diff
@@ -0,0 +1,13 @@
+@mixin define-font($family: null, $size: null, $weight: null) {
+	font: {
+		@if $family {
+			family: $family;
+		}
```

Comment:
Even if we make all properties optional, at least one must be given, otherwise mixin doesn't have a point.
So...If all properties end up optional, we need to add validation when all are null to throw - and cover it in tests.

Reply:
As per our last discussion, we actually and probably want two out of three non-nullable properties.
Check needs to look like 
`@if not ($family and $size and $weight) {
    @include sass-true.error
}`

Reply:
In the meantime, I  added throw.error to ensure that at least two font properties  are provided when using the define-font mixin. I also covered this error message in tests to verify correct behavior.